app/src/main/java/com/google/homesampleapp/screens/settings/SettingsNestedFragment.kt

```diff
@@ -75,6 +76,26 @@ class SettingsNestedFragment :
   }
 
   private fun setupUiElements() {
+    // Alert dialog triggered when the value for "Halfsheet Notification" changes.
+    halfsheetNotificationAlertDialog =
```

Comment:
nit: Does this need to be a class var?

Reply:
showHalfsheetNotificationAlertDialog can be inlined

Reply:
Done.